www/install/steps/process/createDbUser.php

```diff
@@ -63,7 +63,12 @@
 $host = "localhost";
 // if database server is not on localhost...
 if ($parameters['address'] != "127.0.0.1" && $parameters['address'] != "localhost") {
-    $host = explode(" ", $link->getAttribute(PDO::ATTR_CONNECTION_STATUS))[0];
+    $getIpQuery = $link->prepare(
+        'select host from information_schema.processlist WHERE ID=connection_id()'
```

Comment:
Please fix the syntax of the SQL query for consistency (keywords in upper case, identifiers in lower case) and that will be all good for me.

Reply:
done. Thanx